src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.targets

```diff
@@ -25,7 +25,10 @@
     This must be imported before any tools that need to use it are imported.
   -->
   <Import Project="$(MSBuildThisFileDirectory)toolruntime.targets" Condition="'$(ExcludeToolRuntimeImport)' != 'true'"/>
-  
+
+  <!-- import the MicroBuild boot-strapper project if it exists -->
+  <Import Project="$(MSBuildThisFileDirectory)MicroBuild.Core.targets" Condition="Exists('$(MSBuildThisFileDirectory)MicroBuild.Core.targets')" />
```

Comment:
Good point, it should always exist given we're restoring it during init-tools.cmd.  I'll remove the condition.